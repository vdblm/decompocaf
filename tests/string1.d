string s;

int main()
{
    string t;
     s = "hello";
    Print(s);
    t = ReadLine();
    Print(t);
}