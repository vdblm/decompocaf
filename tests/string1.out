hello
CS143 rocks!
